.gitignore

```diff
@@ -1,4 +1,3 @@
 virtualenv/*
-.vscode/settings.json
-config.ini
```

Comment:
Config.ini should remain in the .gitignore file

Reply:
Why? I don't understand.

My motivation in including it was that the system didn't work "out of the box" for lack of a config.ini, yet there was a sample_config.ini.

Reply:
We should add a note in the Readme for the user to copy `sample_config.ini` to `config.ini` the contents of config.ini are only relevant to users with access to our cloud resources.

Reply:
Agreed that if we're not going to include `config.ini` in the repo it should documented. But it seems simpler to just include it in the repo - what's the downside?

Reply:
Listing our service endpoints, even if secure, could potentially be a security target

Reply:
Done.